### PR TITLE
Remove LOGNOTICE and LOGSEVERE from python addons and kodi

### PR DESCRIPTION
**Pull request by phunkyfish**
## Description

Since the introduction of spdlog these have been deprecated. They have already been removed from both all binary addons.

Last step is to remove them for python add-ons and then from kodi.

Forum thread announcing the deprecation is here: https://forum.kodi.tv/showthread.php?tid=353818

## Motivation and Context

No longer required.

## How Has This Been Tested?

On OSX on macbook.

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [X] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by phunkyfish** [comment]
Is there anywhere I should check outside of repo-scripts and repo-plugins?

Here are the references I found in repo-scripts:

LOGSEVERE:

```
script.module.web-pdb/libs/web_pdb/logging.py:        xbmc.log(encode('{}: {}'.format(self._name, msg)), xbmc.LOGSEVERE)
script.module.xbmcswift2/lib/xbmcswift2/logger.py:        'CRITICAL': 'LOGSEVERE',
script.module.xbmcswift2/lib/xbmcswift2/logger.py:        'LOGSEVERE': 5,
script.xbmc.lcdproc/resources/lib/common.py:LOGSEVERE  = xbmc.LOGSEVERE
```

LOGNOTICE:

```
script.metadata.editor/resources/lib/helper.py:NOTICE = xbmc.LOGNOTICE
script.module.addon.signals/lib/AddonSignals.py:        xbmc.log('++++==== script.module.addon.signals: sourceID keyword is DEPRECATED - use source_id ====++++', xbmc.LOGNOTICE)
script.module.dropbox_auth/authenticate.py:    utils.log(str(sys.argv), xbmc.LOGNOTICE)
script.module.dropbox_auth/resources/utils.py:        loglevel: xbmc.LOGDEBUG, xbmc.LOGINFO, xbmc.LOGNOTICE, xbmc.LOGWARNING, xbmc.LOGERROR, xbmc.LOGFATAL
script.module.web-pdb/libs/web_pdb/logging.py:        xbmc.log(encode('{}: {}'.format(self._name, msg)), xbmc.LOGNOTICE)
script.module.xbmcswift2/lib/xbmcswift2/logger.py:        'INFO': 'LOGNOTICE',
script.module.xbmcswift2/lib/xbmcswift2/logger.py:        'LOGNOTICE': 2,
script.plex/lib/kodijsonrpc.py:            xbmc.log(command, xbmc.LOGNOTICE)
script.plex/lib/kodijsonrpc.py:        xbmc.log(command, xbmc.LOGNOTICE)
script.plex/lib/main.py:    xbmc.log('Main: script.plex: REALLY FINISHED', xbmc.LOGNOTICE)
script.plex/lib/util.py:def LOG(msg, level=xbmc.LOGNOTICE):
script.plex/lib/util.py:    xbmc.log('---TEST: {0}'.format(msg), xbmc.LOGNOTICE)
script.plex/lib/windows/kodigui.py:            xbmc.log('kodigui.ManagedControlList._updateItems: Runtime error', xbmc.LOGNOTICE)
script.plex/service.py:        xbmc.log('script.plex: Starting from service (Kiosk Mode)', xbmc.LOGNOTICE)
script.service.checkpreviousepisode/default.py:        ', listen to onAVStarted', xbmc.LOGNOTICE)
script.service.next-episode/libs/logger.py:    log_message(msg, xbmc.LOGNOTICE)
script.xbmc.lcdproc/resources/lib/common.py:LOGNOTICE  = xbmc.LOGNOTICE
script.xbmc.lcdproc/resources/lib/lcdbase.py:          log(LOGNOTICE, "Initialised LCD.xml from defaults")
script.xbmc.lcdproc/resources/lib/lcdbase.py:    log(LOGNOTICE, "Loading settings from %s" % (xmlFile))
script.xbmc.lcdproc/resources/lib/lcdbase.py:    log(LOGNOTICE, "Shutting down")
script.xbmc.lcdproc/resources/lib/lcdproc.py:        log(LOGNOTICE, "Empty driver information reply")
script.xbmc.lcdproc/resources/lib/lcdproc.py:      log(LOGNOTICE, "Driver information reply: " + reply)
script.xbmc.lcdproc/resources/lib/lcdproc.py:        log(LOGNOTICE, "SoundGraph iMON LCD detected")
script.xbmc.lcdproc/resources/lib/lcdproc.py:        log(LOGNOTICE, "Futaba/Targa USB mdm166a VFD detected")
script.xbmc.lcdproc/resources/lib/lcdproc.py:        log(LOGNOTICE, "SoundGraph iMON IR/VFD detected")
script.xbmc.lcdproc/resources/lib/lcdproc.py:      log(LOGNOTICE, "Connected to LCDd at %s:%s, Protocol version %s - Geometry %sx%s characters (%sx%s pixels, %sx%s pixels per character)" % (str(ip), str(port), float(lcdinfo.group(1)), str(self.m_iColumns), str(self.m_iRows), str(self.m_iColumns * self.m_iCellWidth), str(self.m_iRows * self.m_iCellHeight), str(self.m_iCellWidth), str(self.m_iCellHeight)))
service.subtitles.rvm.addic7ed/addic7ed/utils.py:        logger.log(message, xbmc.LOGNOTICE)
service.upnext/resources/lib/utils.py:from xbmc import executeJSONRPC, getRegion, log as xlog, LOGDEBUG, LOGNOTICE
service.upnext/resources/lib/utils.py:    level = LOGDEBUG if debug_logging else LOGNOTICE
service.watchedlist/lib/watchedlist/utils.py:def log(message, loglevel=xbmc.LOGNOTICE):
service.watchedlist/lib/watchedlist/utils.py:        loglevel: xbmc.LOGDEBUG, xbmc.LOGINFO, xbmc.LOGNOTICE, xbmc.LOGWARNING, xbmc.LOGERROR, xbmc.LOGFATAL
weather.gismeteo/resources/libs/simpleplugin.py:        self.log(message, xbmc.LOGNOTICE)
weather.ozweather/resources/lib/common.py:    Log a message at the LOGNOTICE level, i.e. even if Kodi debugging is not turned on. Use sparingly.
weather.ozweather/resources/lib/common.py:    log(message, exception_instance, level=xbmc.LOGNOTICE)
```

And here are the references I found in repo-scripts:

LOGSEVERE:

```
plugin.video.twitch/resources/lib/twitch_addon/addon/common/log_utils.py:from xbmc import LOGDEBUG, LOGERROR, LOGFATAL, LOGINFO, LOGNONE, LOGSEVERE, LOGWARNING  # @UnusedImport
```

LOGNOTICE:

```
plugin.audio.mixcloud/lib/utils.py:            xbmc.log(msg = 'MIXCLOUD ' + message, level = xbmc.LOGNOTICE)
plugin.audio.rne/resources/lib/plugin.py:                xbmc.log(msg=message, level=xbmc.LOGNOTICE)
plugin.audio.rne/resources/lib/plugin.py:                xbmc.log('%s: log this line is not possible due to encoding string problems' % self.plugin_id, level=xbmc.LOGNOTICE)
plugin.audio.rne/resources/lib/plugin.py:                xbmc.log(msg=message, level=xbmc.LOGNOTICE)
plugin.audio.rne/resources/lib/plugin.py:                xbmc.log('%s: _log this line is not possible due to encoding string problems' % self.plugin_id, level=xbmc.LOGNOTICE)
plugin.video.arloview/resources/lib/arlo_stream.py:    def log(self, msg, level=xbmc.LOGNOTICE):
plugin.video.arloview/resources/lib/arlo_stream.py:            if level < xbmc.LOGNOTICE:
plugin.video.arloview/resources/lib/arlo_stream.py:                level = xbmc.LOGNOTICE
plugin.video.arloview/resources/lib/arlo_stream.py:                self.log("Download {} into {}_{}".format(snapshot_type, snapshot_file_prefix, camera['deviceId']),xbmc.LOGNOTICE)
plugin.video.arloview/resources/lib/arlo_stream.py:            self.log("check_first_run()- userid NOT set!", xbmc.LOGNOTICE)
plugin.video.filmsforaction/resources/lib/plugin.py:                xbmc.log(msg=message, level=xbmc.LOGNOTICE)
plugin.video.filmsforaction/resources/lib/plugin.py:                xbmc.log('%s: log this line is not possible due to encoding string problems' % self.plugin_id, level=xbmc.LOGNOTICE)
plugin.video.filmsforaction/resources/lib/plugin.py:                xbmc.log(msg=message, level=xbmc.LOGNOTICE)
plugin.video.filmsforaction/resources/lib/plugin.py:                xbmc.log('%s: _log this line is not possible due to encoding string problems' % self.plugin_id, level=xbmc.LOGNOTICE)
plugin.video.imdb.trailers/resources/lib/imdb_trailers.py:        xbmc.log("[ADD-ON] '{} v{}': {}".format(_plugin, _version, description), xbmc.LOGNOTICE)
plugin.video.twitch/resources/lib/twitch_addon/addon/common/log_utils.py:LOGNOTICE = LOGINFO
plugin.video.twitch/resources/lib/twitch_addon/addon/common/url_dispatcher.py:        log_utils.log('Calling |%s| for mode |%s| with pos args |%s| and kwargs |%s|' % (self.func_registry[mode].__name__, mode, args, kwargs), log_utils.LOGNOTICE)
plugin.video.twitch/resources/lib/twitch_addon/service.py:    log_utils.log('Service: Start', log_utils.LOGNOTICE)
plugin.video.twitch/resources/lib/twitch_addon/service.py:    log_utils.log('Service: Shutdown', log_utils.LOGNOTICE)
```

**Comment by enen92** [comment]
repo-scrapers (Python based scrappers). Would be nice if we could ping the add-on devs of the listed add-ons.

**Comment by phunkyfish** [comment]
Is there an easy way to find out the maintainer for each? The addon.xml for each one?

**Comment by phunkyfish** [comment]
For repo-scrapers

LOGSEVERE:

None

LOGNOTICE:

```
metadata.tvmaze/libs/utils.py:        logger.log(message, xbmc.LOGNOTICE)
```

**Comment by anxdpanic** [comment]
I'll take care of `plugin.video.twitch`

File: xbmc/commons/ilog.h
Comment (enen92) [inline xbmc/commons/ilog.h:20]: There are a few add-ons that make use of the integers/magic numbers directly. Vrt.nu if I am not mistaken does it. @mediaminister @dagwieers et al

Comment (phunkyfish) [reply xbmc/commons/ilog.h:20]: I wonder why they do that. It's not hard to change back if we need to.

**Comment by phunkyfish** [comment]
I could not open an issue / find repos for the following 3 addons:

- script.module.xbmcswift2
- plugin.video.arloview
- plugin.video.imdb.trailers

File: xbmc/commons/ilog.h
Comment (dagwieers) [reply xbmc/commons/ilog.h:20]: We think it is useful for VOD add-ons to make a distinction between:
- No logging (except fatal/error)
- Log network access only (URLs)
- Log everything (caching, menu access, changes, etc)

Comment (phunkyfish) [reply xbmc/commons/ilog.h:20]: Ok, but why does that mean you need to use the integers directly?

File: xbmc/network/httprequesthandler/python/HTTPPythonWsgiInvoker.cpp
Comment (phunkyfish) [inline xbmc/network/httprequesthandler/python/HTTPPythonWsgiInvoker.cpp:30]: Is this correct or should it be LOGWARNING?

File: xbmc/network/httprequesthandler/python/HTTPPythonWsgiInvoker.cpp
Comment (phunkyfish) [inline xbmc/network/httprequesthandler/python/HTTPPythonWsgiInvoker.cpp:57]: Is this correct or should it be LOGWARNING?

File: xbmc/commons/ilog.h
Comment (dagwieers) [reply xbmc/commons/ilog.h:20]: It is defined as integers, so it doesn't really matter.
Mostly to keep log statements brief.

Our log statements are templated, which means the log level needs to be the first argument.
If Python 2.7 disappears we can use f-strings and things will be easier and more concise by nature.

But it shouldn't matter except for the people working on the add-on.

**Comment by phunkyfish** [comment]
I'll plan to merge this in 2 weeks time.

File: xbmc/network/httprequesthandler/python/HTTPPythonWsgiInvoker.cpp
Comment (enen92) [reply xbmc/network/httprequesthandler/python/HTTPPythonWsgiInvoker.cpp:57]: Correct

File: xbmc/network/httprequesthandler/python/HTTPPythonWsgiInvoker.cpp
Comment (Montellese) [reply xbmc/network/httprequesthandler/python/HTTPPythonWsgiInvoker.cpp:30]: Looks ok to me.

**Review by Montellese — APPROVED** [review]
Thanks for cleaning up after me :+1:

**Review by AlwinEsch — APPROVED** [review]
Looks good to me!

**Comment by AlwinEsch** [comment]
Tested the conflict resolf but not good as new commit, Sorry!

On the request by screensaver with doc cleanup this was done. Think to combine the last commit here with the first.

**Comment by phunkyfish** [comment]
> Tested the conflict resolf but not good as new commit, Sorry!
> 
> On the request by screensaver with doc cleanup this was done. Think to combine the last commit here with the first.

No worries, easy to fix up.

**Comment by phunkyfish** [comment]
`repo-scrapers` no longer has any deprecated log statements.

Here is what is left elsewhere.

`repo-scripts`

LOGNOTICE:

```
script.metadata.editor/resources/lib/helper.py:NOTICE = xbmc.LOGNOTICE
script.module.addon.signals/lib/AddonSignals.py:        xbmc.log('++++==== script.module.addon.signals: sourceID keyword is DEPRECATED - use source_id ====++++', xbmc.LOGNOTICE)
script.module.web-pdb/libs/web_pdb/logging.py:        xbmc.log(encode('{}: {}'.format(self._name, msg)), xbmc.LOGNOTICE)
script.module.xbmcswift2/lib/xbmcswift2/logger.py:        'INFO': 'LOGNOTICE',
script.module.xbmcswift2/lib/xbmcswift2/logger.py:        'LOGNOTICE': 2,
script.plex/lib/kodijsonrpc.py:            xbmc.log(command, xbmc.LOGNOTICE)
script.plex/lib/kodijsonrpc.py:        xbmc.log(command, xbmc.LOGNOTICE)
script.plex/lib/main.py:    xbmc.log('Main: script.plex: REALLY FINISHED', xbmc.LOGNOTICE)
script.plex/lib/util.py:def LOG(msg, level=xbmc.LOGNOTICE):
script.plex/lib/util.py:    xbmc.log('---TEST: {0}'.format(msg), xbmc.LOGNOTICE)
script.plex/lib/windows/kodigui.py:            xbmc.log('kodigui.ManagedControlList._updateItems: Runtime error', xbmc.LOGNOTICE)
script.plex/service.py:        xbmc.log('script.plex: Starting from service (Kiosk Mode)', xbmc.LOGNOTICE)
script.service.next-episode/libs/logger.py:    log_message(msg, xbmc.LOGNOTICE)
script.xbmc.lcdproc/resources/lib/common.py:LOGNOTICE  = xbmc.LOGNOTICE
script.xbmc.lcdproc/resources/lib/lcdbase.py:          log(LOGNOTICE, "Initialised LCD.xml from defaults")
script.xbmc.lcdproc/resources/lib/lcdbase.py:    log(LOGNOTICE, "Loading settings from %s" % (xmlFile))
script.xbmc.lcdproc/resources/lib/lcdbase.py:    log(LOGNOTICE, "Shutting down")
script.xbmc.lcdproc/resources/lib/lcdproc.py:        log(LOGNOTICE, "Empty driver information reply")
script.xbmc.lcdproc/resources/lib/lcdproc.py:      log(LOGNOTICE, "Driver information reply: " + reply)
script.xbmc.lcdproc/resources/lib/lcdproc.py:        log(LOGNOTICE, "SoundGraph iMON LCD detected")
script.xbmc.lcdproc/resources/lib/lcdproc.py:        log(LOGNOTICE, "Futaba/Targa USB mdm166a VFD detected")
script.xbmc.lcdproc/resources/lib/lcdproc.py:        log(LOGNOTICE, "SoundGraph iMON IR/VFD detected")
script.xbmc.lcdproc/resources/lib/lcdproc.py:      log(LOGNOTICE, "Connected to LCDd at %s:%s, Protocol version %s - Geometry %sx%s characters (%sx%s pixels, %sx%s pixels per character)" % (str(ip), str(port), float(lcdinfo.group(1)), str(self.m_iColumns), str(self.m_iRows), str(self.m_iColumns * self.m_iCellWidth), str(self.m_iRows * self.m_iCellHeight), str(self.m_iCellWidth), str(self.m_iCellHeight)))
service.subtitles.rvm.addic7ed/addic7ed/utils.py:        logger.log(message, xbmc.LOGNOTICE)
service.upnext/resources/lib/utils.py:from xbmc import executeJSONRPC, getRegion, log as xlog, LOGDEBUG, LOGNOTICE
service.upnext/resources/lib/utils.py:    level = LOGDEBUG if debug_logging else LOGNOTICE
weather.gismeteo/resources/libs/simpleplugin.py:        self.log(message, xbmc.LOGNOTICE)
```

LOGSEVERE:

Note that xbmcswift2 will be updated after PR merge.

```
script.module.web-pdb/libs/web_pdb/logging.py:        xbmc.log(encode('{}: {}'.format(self._name, msg)), xbmc.LOGSEVERE)
script.module.xbmcswift2/lib/xbmcswift2/logger.py:        'CRITICAL': 'LOGSEVERE',
script.module.xbmcswift2/lib/xbmcswift2/logger.py:        'LOGSEVERE': 5,
script.xbmc.lcdproc/resources/lib/common.py:LOGSEVERE  = xbmc.LOGSEVERE
```

`repo-plugins`

LOGNOTICE:

```
plugin.audio.mixcloud/lib/utils.py:            xbmc.log(msg = 'MIXCLOUD ' + message, level = xbmc.LOGNOTICE)
plugin.video.arloview/resources/lib/arlo_stream.py:    def log(self, msg, level=xbmc.LOGNOTICE):
plugin.video.arloview/resources/lib/arlo_stream.py:            if level < xbmc.LOGNOTICE:
plugin.video.arloview/resources/lib/arlo_stream.py:                level = xbmc.LOGNOTICE
plugin.video.arloview/resources/lib/arlo_stream.py:                self.log("Download {} into {}_{}".format(snapshot_type, snapshot_file_prefix, camera['deviceId']),xbmc.LOGNOTICE)
plugin.video.arloview/resources/lib/arlo_stream.py:            self.log("check_first_run()- userid NOT set!", xbmc.LOGNOTICE)
plugin.video.imdb.trailers/resources/lib/imdb_trailers.py:        xbmc.log("[ADD-ON] '{} v{}': {}".format(_plugin, _version, description), xbmc.LOGNOTICE)
plugin.video.twitch/resources/lib/twitch_addon/addon/common/log_utils.py:LOGNOTICE = LOGINFO
plugin.video.twitch/resources/lib/twitch_addon/addon/common/url_dispatcher.py:        log_utils.log('Calling |%s| for mode |%s| with pos args |%s| and kwargs |%s|' % (self.func_registry[mode].__name__, mode, args, kwargs), log_utils.LOGNOTICE)
plugin.video.twitch/resources/lib/twitch_addon/service.py:    log_utils.log('Service: Start', log_utils.LOGNOTICE)
plugin.video.twitch/resources/lib/twitch_addon/service.py:    log_utils.log('Service: Shutdown', log_utils.LOGNOTICE)
```

LOGSEVERE:

```
plugin.video.twitch/resources/lib/twitch_addon/addon/common/log_utils.py:from xbmc import LOGDEBUG, LOGERROR, LOGFATAL, LOGINFO, LOGNONE, LOGSEVERE, LOGWARNING  # @UnusedImport
```

**Comment by phunkyfish** [comment]
Plan is to merge this PR on Friday this week.

**Comment by anxdpanic** [comment]
https://github.com/xbmc/repo-plugins/pull/3105 Good to go for Twitch, appreciate the heads up 👍

**Comment by phunkyfish** [comment]
I will merge this PR later today.

**Comment by CvH** [comment]
@phunkyfish that pr breaks building for RPi4 (likely others that use too that use DVDVideoCodecDRMPRIME), reverting the pr fixed building at current master https://github.com/xbmc/xbmc/commit/1cd618c9cfc9a6c57474959a77ae6a8db3013259

```
[1321/1557] Building CXX object build/cores/VideoPlayer/codecs/video/CMakeFiles/dvdvideocodecs.dir/DVDVideoPPFFmpeg.cpp.o
[1322/1557] Building CXX object build/cores/VideoPlayer/codecs/video/CMakeFiles/dvdvideocodecs.dir/DVDVideoCodecDRMPRIME.cpp.o
FAILED: build/cores/VideoPlayer/codecs/video/CMakeFiles/dvdvideocodecs.dir/DVDVideoCodecDRMPRIME.cpp.o 
/LE/build.LibreELEC-RPi4.arm-9.80-devel/toolchain/bin/ccache /LE/build.LibreELEC-RPi4.arm-9.80-devel/toolchain/bin/armv8a-libreelec-linux-gnueabihf-g++  -DHAS_NEON -I../ -I../lib -I../xbmc -I../xbmc/platform/linux -I../xbmc/cores/VideoPlayer -Ibuild -Ibuild/include -I../xbmc/platform/posix -I/LE/build.LibreELEC-RPi4.arm-9.80-devel/toolchain/armv8a-libreelec-linux-gnueabihf/sysroot/usr/include/dbus-1.0 -I/LE/build.LibreELEC-RPi4.arm-9.80-devel/toolchain/armv8a-libreelec-linux-gnueabihf/sysroot/usr/lib/dbus-1.0/include -I/LE/build.LibreELEC-RPi4.arm-9.80-devel/toolchain/armv8a-libreelec-linux-gnueabihf/sysroot/usr/include/python3.8 -I/LE/build.LibreELEC-RPi4.arm-9.80-devel/toolchain/armv8a-libreelec-linux-gnueabihf/sysroot/usr/include/samba-4.0 -I/LE/build.LibreELEC-RPi4.arm-9.80-devel/toolchain/armv8a-libreelec-linux-gnueabihf/sysroot/usr/include/libxml2 -Ibuild/cores/RetroPlayer/messages -I/LE/build.LibreELEC-RPi4.arm-9.80-devel/toolchain/armv8a-libreelec-linux-gnueabihf/sysroot/usr/include/freetype2 -I/LE/build.LibreELEC-RPi4.arm-9.80-devel/toolchain/armv8a-libreelec-linux-gnueabihf/sysroot/usr/include/fribidi -Ibuild/libdvd/include -I/LE/build.LibreELEC-RPi4.arm-9.80-devel/toolchain/armv8a-libreelec-linux-gnueabihf/sysroot/usr/include/lzo -I/LE/build.LibreELEC-RPi4.arm-9.80-devel/toolchain/armv8a-libreelec-linux-gnueabihf/sysroot/usr/include/libdrm -march=armv8-a+crc -mtune=cortex-a53 -mabi=aapcs-linux -Wno-psabi -Wa,-mno-warn-deprecated -mfloat-abi=hard -mfpu=neon-fp-armv8 -Wall -pipe  -O3 -fomit-frame-pointer -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11 -D__GBM__=1 -DPLATFORM_SETTINGS_FILE=gbm.xml -Os -DNDEBUG -flto=16 -fno-fat-lto-objects   -DTARGET_POSIX -DTARGET_LINUX -D_GNU_SOURCE -DHAVE_LINUX_UDMABUF=1 -DHAVE_LINUX_DMA_HEAP=1 -DHAVE_LINUX_DMA_BUF=1 -DHAVE_MKOSTEMP=1 -DHAVE_LINUX_MEMFD=1 -DHAVE_STATX=1 -D__STDC_CONSTANT_MACROS -D_FILE_OFFSET_BITS=64 -DHAS_POSIX_NETWORK -DHAS_LINUX_NETWORK -DHAS_BUILTIN_SYNC_ADD_AND_FETCH=1 -DHAS_BUILTIN_SYNC_SUB_AND_FETCH=1 -DHAS_BUILTIN_SYNC_VAL_COMPARE_AND_SWAP=1 -DHAVE_INOTIFY=1 -DHAVE_POSIX_FADVISE=1 -DHAVE_LOCALTIME_R=1 -DHAVE_GMTIME_R=1 -DHAVE_INTTYPES_H=1 -DHAS_ALSA=1 -DHAS_AVAHI=1 -DHAS_ZEROCONF=1 -DHAVE_LIBBLURAY=1 -DHAVE_LIBBLURAY_BDJ=1 -DHAVE_LIBCEC=1 -DHAS_DBUS=1 -DHAS_ISO9660PP=1 -DHAS_LIRC=1 -DHAS_WEB_SERVER=1 -DHAS_WEB_INTERFACE=1 -DHAS_FILESYSTEM_NFS=1 -DHAS_NFS_SET_TIMEOUT -DHAS_AIRPLAY=1 -DHAS_PULSEAUDIO=1 -DHAS_PYTHON=1 -DHAS_FILESYSTEM_SMB=1 -DHAVE_LIBUDEV=1 -DHAS_UDFREAD=1 -DHAVE_LIBXSLT=1 -DFFMPEG_VER_SHA=\"4.3\" -I/LE/build.LibreELEC-RPi4.arm-9.80-devel/toolchain/armv8a-libreelec-linux-gnueabihf/sysroot/usr/include/fribidi -DSPDLOG_FMT_EXTERNAL -DSPDLOG_DEBUG_ON -DSPDLOG_NO_ATOMIC_LEVELS -DSPDLOG_ENABLE_PATTERN_PADDING -DHAS_EGL=1 -DHAVE_GBM=1 -DHAS_GBM_BO_MAP=1 -DHAS_GBM_MODIFIERS=1 -DHAS_GLES=3 -DHAS_MARIADB=1 -DHAS_UPNP=1 -DHAS_DVD_DRIVE -DHAS_CDDA_RIPPER -DHAS_AIRTUNES=1 -DBIN_INSTALL_PATH=\"/usr/lib/kodi\" -DINSTALL_PATH=\"/usr/share/kodi\" -std=c++14 -MD -MT build/cores/VideoPlayer/codecs/video/CMakeFiles/dvdvideocodecs.dir/DVDVideoCodecDRMPRIME.cpp.o -MF build/cores/VideoPlayer/codecs/video/CMakeFiles/dvdvideocodecs.dir/DVDVideoCodecDRMPRIME.cpp.o.d -o build/cores/VideoPlayer/codecs/video/CMakeFiles/dvdvideocodecs.dir/DVDVideoCodecDRMPRIME.cpp.o -c ../xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecDRMPRIME.cpp
../xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecDRMPRIME.cpp: In member function 'virtual bool CDVDVideoCodecDRMPRIME::Open(CDVDStreamInfo&, CDVDCodecOptions&)':
../xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecDRMPRIME.cpp:226:15: error: 'LOGNOTICE' was not declared in this scope; did you mean 'LOGNONE'?
  226 |     CLog::Log(LOGNOTICE, "CDVDVideoCodecDRMPRIME::{} - codec {} disallowed",
      |               ^~~~~~~~~
      |               LOGNONE
[1323/1557] Building CXX object build/cores/VideoPlayer/inputstreams/CMakeFiles/dvdinputstreams.dir/DVDFactoryInputStream.cpp.o
[1324/1557] Building CXX object build/cores/VideoPlayer/codecs/video/CMakeFiles/dvdvideocodecs.dir/DVDVideoCodecFFmpeg.cpp.o
../xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecFFmpeg.cpp: In member function 'virtual bool CDVDVideoCodecFFmpeg::GetPictureCommon(VideoPicture*)':
../xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecFFmpeg.cpp:999:78: warning: 'int8_t* av_frame_get_qp_table(AVFrame*, int*, int*)' is deprecated [-Wdeprecated-declarations]
  999 |                                                   &pVideoPicture->qscale_type);
      |                                                                              ^
In file included from /LE/build.LibreELEC-RPi4.arm-9.80-devel/toolchain/armv8a-libreelec-linux-gnueabihf/sysroot/usr/include/libavcodec/avcodec.h:38,
                 from ../xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemux.h:34,
                 from ../xbmc/cores/VideoPlayer/DVDStreamInfo.h:11,
                 from ../xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecFFmpeg.h:12,
                 from ../xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecFFmpeg.cpp:9:
/LE/build.LibreELEC-RPi4.arm-9.80-devel/toolchain/armv8a-libreelec-linux-gnueabihf/sysroot/usr/include/libavutil/frame.h:725:9: note: declared here
  725 | int8_t *av_frame_get_qp_table(AVFrame *f, int *stride, int *type);
      |         ^~~~~~~~~~~~~~~~~~~~~
[1325/1557] Building CXX object build/cores/VideoPlayer/demuxers/CMakeFiles/dvddemuxers.dir/DemuxMultiSource.cpp.o
[1326/1557] Building CXX object build/cores/VideoPlayer/demuxers/CMakeFiles/dvddemuxers.dir/DVDDemuxUtils.cpp.o
[1327/1557] Building CXX object build/cores/VideoPlayer/demuxers/CMakeFiles/dvddemuxers.dir/DVDDemuxClient.cpp.o
[1328/1557] Building CXX object build/cores/VideoPlayer/demuxers/CMakeFiles/dvddemuxers.dir/DVDFactoryDemuxer.cpp.o
[1329/1557] Building CXX object build/cores/VideoPlayer/demuxers/CMakeFiles/dvddemuxers.dir/DVDDemuxFFmpeg.cpp.o
[1330/1557] Building CXX object build/cores/VideoPlayer/inputstreams/CMakeFiles/dvdinputstreams.dir/DVDInputStreamFile.cpp.o
[1331/1557] Building CXX object build/cores/VideoPlayer/inputstreams/CMakeFiles/dvdinputstreams.dir/DVDInputStreamStack.cpp.o
[1332/1557] Building CXX object build/cores/VideoPlayer/inputstreams/CMakeFiles/dvdinputstreams.dir/InputStreamMultiSource.cpp.o
[1333/1557] Building CXX object build/cores/VideoPlayer/inputstreams/CMakeFiles/dvdinputstreams.dir/DVDInputStreamNavigator.cpp.o
[1334/1557] Building CXX object build/cores/VideoPlayer/inputstreams/CMakeFiles/dvdinputstreams.dir/InputStreamAddon.cpp.o
[1335/1557] Building CXX object build/cores/VideoPlayer/inputstreams/CMakeFiles/dvdinputstreams.dir/InputStreamPVRChannel.cpp.o
[1336/1557] Building CXX object build/cores/VideoPlayer/inputstreams/CMakeFiles/dvdinputstreams.dir/InputStreamPVRRecording.cpp.o
[1337/1557] Building CXX object build/cores/VideoPlayer/inputstreams/CMakeFiles/dvdinputstreams.dir/InputStreamPVRBase.cpp.o
ninja: build stopped: subcommand failed.
```

**Comment by phunkyfish** [comment]
Let me check. Maybe I missed a LOGNOTICE reference.

**Comment by phunkyfish** [comment]
That file in the source tree does not contain that statement. Do you need to update the source?

**Comment by CvH** [comment]
@phunkyfish sorry for false alarm, we hold some patch that needed a change to align to the LOGNOTICE removal

**Comment by phunkyfish** [comment]
No worries, glad you have it sorted out.

**Pull request merged**
